renderlib/io/CMakeLists.txt

```diff
@@ -28,22 +28,12 @@ add_subdirectory(libCZI)
 
 # end libczi dependency
 
-# zstd dependency for tensorstore / riegeli
-FetchContent_Declare(
-	zstd
-	URL "https://github.com/facebook/zstd/archive/v1.5.6.tar.gz"
-	URL_HASH SHA256=8c29e06cf42aacc1eafc4077ae2ec6c6fcb96a626157e0593d5e82a34fd403c1
-)
-
-# Additional FetchContent_Declare calls as needed...
-FetchContent_MakeAvailable(zstd)
-
 # tensorstore dependency for renderlib
 set(TENSORSTORE_USE_SYSTEM_TIFF ON)
 FetchContent_Declare(
 	tensorstore
-	URL "https://github.com/google/tensorstore/archive/refs/tags/v0.1.58.tar.gz"
-	URL_HASH SHA256=e916c9dc0ca9a7f272fc975b76ea06c452f921e65a8b201fc45b4ea96261a6ad
+	GIT_REPOSITORY https://github.com/google/tensorstore.git
+	GIT_TAG v0.1.61
```

Comment:
The changes in this file are a throw-in to fix builds on github-actions, separate from the intent of the PR.